recipe/meta.yaml

```diff
@@ -10,36 +10,37 @@ source:
   sha256: e78a116feb2ebd04de31a8d8707c65e8e15a64aa8999a40fea305e3909bd6533
   patches:
     - include_libxml_sax.patch
+    - nexus_windows.patch  # [win]
 
 build:
-  number: 2
-  skip: true    # [win]
+  number: 3
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - cmake >=2.8.8
-    - make
+    - ninja
   host:
-    - hdf5
+    - hdf5 
```

Comment:
Could you please remove the trailing white space to avoid this change. Sorry to be picky :-)